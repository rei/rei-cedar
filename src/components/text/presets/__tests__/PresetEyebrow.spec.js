import { mount } from '../../../../../test/vue-jest-style-workaround.js';
import PresetEyebrow from '../PresetEyebrow.vue';


describe('PresetEyebrow', () => {
  describe('default configuration', () => {
    let wrapper;
    beforeEach(() => {
      wrapper = mount(PresetEyebrow, {
        slots: {
          default: 'paragraph'
        }
      });
    });
    it('renders correctly', () => {
      expect(wrapper.element).toMatchSnapshot();
    });
  });

});
