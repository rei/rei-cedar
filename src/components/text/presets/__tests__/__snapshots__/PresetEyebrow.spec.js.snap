// Vitest Snapshot v1

exports[`PresetEyebrow > default configuration > renders correctly 1`] = `
<p
  class="cdr-text cdr-eyebrow"
>
  <!-- @slot CdrText content -->
  
  
  paragraph
  
  
</p>
`;
